test/functional/test_framework/blocktools.py

```diff
@@ -217,3 +219,11 @@ def send_to_witness(use_p2wsh, node, utxo, pubkey, encode_p2sh, amount, sign=Tru
             tx_to_witness = ToHex(tx)
 
     return node.sendrawtransaction(tx_to_witness)
+
+class TestFrameworkScript(unittest.TestCase):
+    def test_createcoinbase(self):
```

Comment:
nit: I would keep the underscore in between, i.e. `test_create_coinbase`, and maybe just call it `height` since it's not in a sequence of blocks.